src/relay/pass/pattern_util.h

```diff
@@ -415,6 +415,71 @@ static inline Expr Full(Expr fill_value,
   return CallNode::make(op, {fill_value}, Attrs(attrs), {});
 }
 
+static inline Expr Conv2D(Expr data, Expr weight, Array<IndexExpr> strides,
```

Comment:
It looks this is the same as `MakeConv2d`, right?
If this is true, should we just keep one signature instead of having duplication. I am not strongly against this because it's obviously used by other cases as well.

Reply:
Yeah, I kept it to follow other usecases. I guess this repetition might be because typically TVM does not want header and implementation linking problems. Will keep it Conv2D for now.

Reply:
I don't have a strong feeling about this, actually I'm not sure why we prefer to copying this (and the others) here instead of adding declarations

Reply:
I think linking should be fine. We can put TVM_DLL if needed. But anyway, we can keep it this way for now.